share/agrammon.openapi

```diff
@@ -3,6 +3,62 @@ info:
     version: 1.0.0,
     title: OpenApi Agrammon,
 paths:
+    /create_account:
+        post:
+            summary: Create new user account
+            operationId: createAccount
+            requestBody:
+                required: true
+                content:
+                    application/json:
+                        schema:
+                            type: object
+                            required:
+                                - email
+                                - password
+                            properties:
+                                email:
+                                    description: User's email used as username
+                                    type: string
+                                password:
+                                    description: Inital password
+                                    type: string
+                                key:
+                                    description: Validation key
+                                    type: string
+                                firstname:
+                                    description: Firstname
+                                    type: string
+                                lastname:
+                                    description: Lastname
+                                    type: string
+                                org:
+                                    description: Organization
+                                    type: string
+                                role:
+                                    description: User role
+                                    type: string
+                                    enum:
+                                        - admin
+                                        - user
+            responses:
+                '200':
+                    description: Account created.
+                    content:
+                        application/json:
+                            schema:
+                                type: object
+                                required:
+                                    - username
+                                properties:
+                                    username:
+                                        type: string
+                '409':
+                    description: User already exists
```

Comment:
And this only documents the conflict case too

Reply:
I added a few more status codes.